src/main/resources/include/hail/NativeStatus.h

```diff
@@ -6,9 +6,10 @@
 #include <cstdio>
 #include <string>
 
-namespace hail {
+// NativeStatus is not in the "hail" namespace, because that would complicate
+// the mangled names of functions which have a "NativeStatus*" parameter
```

Comment:
Functions called from JVM  are extern "C", so they are not mangled.  Is this necessary?

Reply:
It is necessary because we pass a NativeStatus* as the first parameter to
each (generated) LongFunc, and those funcs need to have namespace-
qualified C++ symbols, and thus also get the arg types mangled into the symbol name.  If the "NativeStatus*" arg type is itself qualified-and-mangled, that gets a little confusing.